modules/world/map/local_map.cpp

```diff
@@ -28,6 +28,20 @@ LaneId LocalMap::GoalLaneIdFromGoalDefinitionPolygon(const GoalDefinitionPolygon
   return LaneId(0);
 }
 
+LanePtr LocalMap::FindLane(const Point2d& point) const {
+  LanePtr lane = map_interface_->FindLane(point);
+  return lane;
+}
+
+bool LocalMap::hasCorrectDrivingDirection(const State& state) const {
+  geometry::Point2d position(state(models::dynamic::StateDefinition::X_POSITION), state(models::dynamic::StateDefinition::Y_POSITION));
```

Comment:
Not longer than 80 lines; 

using models::dynamic::StateDefinition::X_POSITION;

above